src/compiler/Fork.js

```diff
@@ -50,6 +50,13 @@ module.exports = class Fork extends EventEmitter {
         }
 
         forks[platformMatch[1]].setSocket(socket);
+
+        socket.on('error', err => {
+          this.emit(Events.BUILD_FAILED, {
+            message: `Socket: ${err.toString()}`,
+          });
+          throw err;
```

Comment:
Don't think anybody can catch this error. Should we throw it?

Reply:
It will be written in `.artifacts`